docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+.. inclusion-marker-start-do-not-remove
+
+Building a Conda environment with GPU support for Horovod
+=========================================================
+
+In this section we describe how to build Conda environments for my deep learning projects using 
+Horovod to enable distributed training across multiple GPUs (either on the same node or 
+spread across multuple nodes).
+
+Installing the NVIDIA CUDA Toolkit
+----------------------------------
+
+First thing you need to do is to install the `appropriate version`_ of the NVIDIA CUDA Toolkit on 
+your workstation. In this section we are using `NVIDIA CUDA Toolkit 10.1`_ (`documentation`_) 
+which works with all three deep learning frameworks that are currently supported by Horovod.
+
+Why not just use the :code:`cudatoolkit` package?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typically when installing PyTorch, TensorFlow, or Apache MXNet with GPU support using Conda you 
+simply add the appropriate version of the :code:`cudatoolkit` package to your 
+:code:`environment.yml` file. Unfortunately, for the moment at least, the cudatoolkit packages 
+available via Conda do not include the `NVIDIA CUDA Compiler (NVCC)`_ which is required in order 
+to build Horovod extensions for PyTorch, TensorFlow, or MXNet.
+
+What about the :code:`cudatoolkit-dev` package?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While there are :code:`cudatoolkit-dev` packages available from :code:`conda-forge` that do 
+include NVCC, we have had difficult getting these packages to consistently install properly. Some 
+of the available builds require manual intervention to accept license agreements making these 
+builds unsuitable for installing on remote systems (which is critical functionality). Other builds 
+seems to work on Ubuntu but not on other flavors of Linux.
+
+Despite this we would encourage you to try adding :code:`cudatoolkit-dev` to your 
+:code:`environment.yml` file and see what happens! The package is well maintained so perhaps it 
+will become more stable in the future.
+
+Use the :code:`nvcc_linux-64` meta-package
```

Comment:
I recommend using double backquotes (``) for code highlighting

Reply:
Done